products/quarkus.md

```diff
@@ -68,7 +68,7 @@ releases:
 
 -   releaseCycle: "2.16"
     releaseDate: 2023-01-25
-    eol: false
+    eol: 2023-10-31
```

Comment:
Please provide the source for this date. On https://quarkus.io/security/#supported-versions Latest 2.x is listed as Supported.

Reply:
Well, the source is me :). I'm in the process of updating our website and repository too.

Reply:
(I'm the release manager for Quarkus)

Reply:
BTW, we need to improve and have also proper EOL dates on our side.

Reply:
Also, sorry, I should have stated that I was from the Quarkus team.

Reply:
Allright, glad to read that you are pushing the details to this project first hand :handshake: